README.md

```diff
@@ -1 +1,58 @@
 # Metabroker
+
+Metabroker is an implementation of a service broker for Kubernetes. It provides
+a framework for creating service offerings and plans for provisioning Helm
+charts as service instances.
+
+## Trying Metabroker locally
+
+1. Start Minikube:
+
+```shell
+minikube start --cpus 4 --embed-certs
+```
+
+2. Install the CRDs:
+
+```shell
+(cd "$(git rev-parse --show-toplevel)/apis/"; make install-crd)
```

Comment:
```suggestion
make -C "$(git rev-parse --show-toplevel)/apis/" install-crd
```
Alternatively, assume that `$PWD` is the source directory and people can figure out what you mean (and drop the `git rev-parse` magic).
Same for the other commands.

Reply:
Addressed in 48ab47a.